packages/gen-ai-hub/src/index.ts

```diff
@@ -1,17 +1,10 @@
+export * from './client/index.js';
```

Comment:
Is this fine?

Reply:
As long as no checks fail, yes. I think the API check is not in place yet, but once it is this might cause problems, not sure. I'd suggest to fix it then if needed.